### DISCUSSION TITLE
basic image rendering not working on localhost

### DISCUSSION THREAD

**Discussion by quadsurf** [vercel/next.js#13515] [Help]
Static file serving API does not work as you explain it here:
https://nextjs.org/docs/basic-features/static-file-serving

how does it not work?
- it displays the browser's default broken image icon

reproduction steps...
- i place image in /public sub-dir
- i call it using regular html <img element tag
- i reboot the local server after adding image to /pages jsx files

I'm on 9.3.4 so I'm not using /static  nor  /public/static

I even tried hard-pathing it to  '../../public/test-image.jpg' (I knew this wouldn't work, but i had to at least try it out any way)

for some strange reason though, the  /public sub-dir is not serving files

DO I NEED WEBPACK LOADERS???

the exact error is this:
`Failed to compile`
`./pages/index.jsx`
`Module not found: Can't resolve './test-image.jpg' in '/Users/home/projects/next-apps/project1/src/pages'`
This could be the problem... if you look at the path above, it's resolving to the /pages sub-dir instead of the /public folder

**Comment by rafaelalmeidatk** [comment]
Are you using `require` to load the image? You need to set the image path as a regular string:
```
<img src="/test-image.jpg" alt="test image" />
```
